pkg/ingestor/parser/cyclonedx/parser_cyclonedx.go

```diff
@@ -132,7 +137,15 @@ func (c *cyclonedxParser) getPackages(cdxBom *cdx.BOM) error {
 			// the required purl for package node. Currently there is no use-case
 			// to capture OS for GUAC.
 			if comp.Type != cdx.ComponentTypeOS {
-				pkg, err := asmhelpers.PurlToPkg(comp.PackageURL)
+				purl := comp.PackageURL
```

Comment:
added to address issue https://github.com/guacsec/guac/issues/828. Need to add some tests.